ui/app/components/app/confirmation-footer/confirmation-footer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Button from '../../ui/button';
+
+export default function ConfirmationFooter({
+  onApprove,
+  onCancel,
+  approveText,
+  cancelText,
+  alerts,
+}) {
+  const className = classnames('confirmation-footer', {
+    'confirmation-footer--with-alerts':
+      alerts && Object.keys(alerts).length > 0,
+  });
+
+  return (
+    <div className={className}>
+      {alerts && alerts}
```

Comment:
is `alerts && alerts` needed? is `{alerts}` sufficient?

Reply:
resolved in latest commit